pom.xml

```diff
@@ -111,69 +111,68 @@
     <maven-clean.version>3.1.0</maven-clean.version>
     <maven-compiler.version>3.8.1</maven-compiler.version>
     <maven-dependency.version>3.1.1</maven-dependency.version>
-    <maven-deploy.version>2.8.2</maven-deploy.version>
+    <maven-deploy.version>3.0.0-M1</maven-deploy.version>
     <maven-download.version>1.4.2</maven-download.version>
     <maven-eclipse.version>2.10</maven-eclipse.version>
-    <maven-enforcer.version>3.0.0-M2</maven-enforcer.version>
+    <maven-enforcer.version>3.0.0-M3</maven-enforcer.version>
     <maven-gpg.version>1.6</maven-gpg.version>
-    <maven-install.version>2.5.2</maven-install.version>
+    <maven-install.version>3.0.0-M1</maven-install.version>
     <maven-jacoco.version>0.8.5</maven-jacoco.version>
-    <maven-jar.version>3.1.2</maven-jar.version>
+    <maven-jar.version>3.2.0</maven-jar.version>
     <maven-javadoc.version>3.1.1</maven-javadoc.version>
     <maven-jxr.version>3.0.0</maven-jxr.version>
     <maven-pmd.version>3.12.0</maven-pmd.version>
     <maven-project-info.version>3.0.0</maven-project-info.version>
     <maven-release.version>2.5.3</maven-release.version>
     <maven-resources.version>3.1.0</maven-resources.version>
     <maven-shade.version>3.2.1</maven-shade.version>
-    <maven-site.version>3.7.1</maven-site.version>
+    <maven-site.version>3.8.2</maven-site.version>
     <maven-source.version>3.1.0</maven-source.version>
-    <maven-surefire.version>2.22.2</maven-surefire.version>
-    <metrics.version>4.1.1</metrics.version>
-    <mockito.version>3.1.0</mockito.version>
+    <maven-surefire.version>3.0.0-M4</maven-surefire.version>
+    <metrics.version>4.1.2</metrics.version>
+    <mockito.version>3.2.4</mockito.version>
     <mssql.version>7.4.1.jre8</mssql.version>
     <multithreadedtc.version>1.01</multithreadedtc.version>
-    <mysql.version>8.0.18</mysql.version>
+    <mysql.version>8.0.19</mysql.version>
     <nexus-staging-maven.version>1.6.8</nexus-staging-maven.version>
     <node.version>v12.13.1</node.version> <!-- https://nodejs.org/en/download/releases/ -->
     <npm.version>6.13.1</npm.version> <!-- https://github.com/npm/cli/releases -->
     <nitor-matchers.version>1.3</nitor-matchers.version>
-    <postgresql.version>42.2.8</postgresql.version>
+    <postgresql.version>42.2.9</postgresql.version>
     <postgresql-embedded.version>2.10</postgresql-embedded.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <reactor.netty.version>0.9.1.RELEASE</reactor.netty.version>
-    <!-- 0.9.11 not used because https://trello.com/c/ZEcAoiSq/406-remove-05-mb-of-exceptions-farted-out-when-starting-nflow-jetty -->
-    <reflections.version>0.9.11</reflections.version>
-    <slf4j.version>1.7.29</slf4j.version>
+    <reactor.netty.version>0.9.3.RELEASE</reactor.netty.version>
+    <reflections.version>0.9.12</reflections.version>
```

Comment:
Problem introduced in 0.9.11 still exists. Revert back to 0.9.10 and restore & update the comment also.